lib/Conversion/ImportVerilog/Structure.cpp

```diff
@@ -722,7 +725,6 @@ Context::convertModuleBody(const slang::ast::InstanceBodySymbol *module) {
   builder.setInsertionPointToEnd(lowering.op.getBody());
 
   // Convert the body of the module.
-  ValueSymbolScope scope(valueSymbols);
   for (auto &member : module->members()) {
     auto loc = convertLocation(member.location);
     if (failed(member.visit(ModuleVisitor(*this, loc))))
```

Comment:
From my perspective, when we create `scope` at traversing the `members` of the `instanceBody`, the variables stored int `valueSymbols` cannot be used by hierarchical name. An error like 
```
/home/phoenix/work/HDLBits/test09.sv:19:15: error: unknown name `data`
  assign w0 = sub1.data;
              ^
/home/phoenix/work/HDLBits/test09.sv:4:15: note: no value generated for Variable
  logic [7:0] data;
```
However, I moved the location of creating `scope` to the top/outermost level, and this error still exists.
I don't know how to deal with it. I need @fabianschuiki help. Thanks in advance :smiley:!

Reply:
For the above example, we have the top module `m`, and submodule `m1`(instance `n`). I think we should create `scope` to contain 
```
        "name": "m",
        "kind": "InstanceBody",
        "addr": 2199025897488,
        "members": [
          {
            "name": "n",
            "kind": "Instance",
         .......
```
, then we can handle hierarchical values. But now, we will create `scope` when we visit `InstanceBody`.

Reply:
Maybe should we need a new op to represent the hierarchical name, rather than get it from `valueSymbols`?